From 0cc59adf93dba68fc08d736e69d7635d0e595291 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Fri, 24 Mar 2023 17:25:59 +0100
Subject: [PATCH] hwdb: drop boilerplate about match patterns in two more cases

Follow-up for adbe000e3df5f91572bf4a77a2acd84ff5b87320.
Somehow I missed those two files.

(cherry picked from commit 61b69bec8e37f4316019a8fba4ccb5af3470b72f)

Related: RHEL-5950
---
 hwdb.d/70-joystick.hwdb | 5 -----
 hwdb.d/70-touchpad.hwdb | 5 -----
 2 files changed, 10 deletions(-)

diff --git a/hwdb.d/70-joystick.hwdb b/hwdb.d/70-joystick.hwdb
index 9d5c4fc069..8e942c8c1b 100644
--- a/hwdb.d/70-joystick.hwdb
+++ b/hwdb.d/70-joystick.hwdb
@@ -5,11 +5,6 @@
 # The lookup keys are composed in:
 #   70-joystick.rules
 #
-# Note: The format of the "joystick:" prefix match key is a
-# contract between the rules file and the hardware data, it might
-# change in later revisions to support more or better matches, it
-# is not necessarily expected to be a stable ABI.
-#
 # Match string format:
 # joystick:<bustype>:v<vid>p<pid>:name:<name>:
 #
diff --git a/hwdb.d/70-touchpad.hwdb b/hwdb.d/70-touchpad.hwdb
index 8194d98358..262bca3a19 100644
--- a/hwdb.d/70-touchpad.hwdb
+++ b/hwdb.d/70-touchpad.hwdb
@@ -5,11 +5,6 @@
 # The lookup keys are composed in:
 #   70-touchpad.rules
 #
-# Note: The format of the "touchpad:" prefix match key is a
-# contract between the rules file and the hardware data, it might
-# change in later revisions to support more or better matches, it
-# is not necessarily expected to be a stable ABI.
-#
 # Match string format:
 # touchpad:<subsystem>:v<vid>p<pid>:name:<name>:
 #
